docker-compose.yml

```diff
@@ -1,18 +1,22 @@
 web:
   build: .
+  dockerfile: Dockerfile.development
   volumes:
     - .:/champaign
```

Comment:
Also, I might be wrong, but there’s little benefit to sync cache and temporary files with host’s volume. Should they be inspected for whatever reason, one could easily `docker-compose exec web bash` a running container. 
Thus, I’d add these entries:
```
    - /champaign/node_modules/
    - /champaign/tmp/
    - /champaign/public/packs/
    - /champaign/public/packs-test/
    - /public/system/
```
Plus they won’t interfere anymore with any previous local build.